### PR TITLE
Backward compatibility with vim-slime and documentation for legacy features and upgrade to new vim-slime.

### PR DESCRIPTION
**Pull request by Klafyvel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,24 @@ This fork uses plugins to send data to its targets.
...
+[vim-slime-cells](https://github.com/Klafyvel/vim-slime-cells/).
+
+You will need a plugin for your specific target. Please see [available target plugins](#available-plugins). 
+Please refer to the target plugin documentation and [Registering the plugin](#registering-the-plugin) 
+below for configuration.
+
+## Available plugins
+
+Here are the available target plugins. If you write a new plugin for your
+favourite target, please open a [pull request](https://github.com/jpalardy/vim-slime-ext-plugins/compare).
```
Comment (jpalardy) [inline README.md]: compare? (doesn't look right)

File: doc/vim-slime.txt [hunk 2]
```diff
@@ -56,10 +56,9 @@ vim-slime to send data to the REPL. The signature must be :
 `MyTargetFunction(config, text)`
 
 With `config` being the configuration variable and `text` the text which is to
-be sent. It is a good practice that `config` is a dictionary, and that a
-plugins register their configuration in `config["myplugin"]`. Vim-slime takes
-care to initialize the configuration by calling the defined configuration
-function. Such a function must have the following signature :
+be sent. Vim-slime takescare to initialize the configuration by calling the
```
Comment (jpalardy) [inline doc/vim-slime.txt]: takescare -> takes care

File: README.md (hunk 1 above)
Comment (Klafyvel) [reply README.md]: That's the URL the "open pull request" button points to :thinking:

Comment (Klafyvel) [reply README.md]: I'll replace with https://github.com/jpalardy/vim-slime-ext-plugins/edit/main/README.md

Comment (Klafyvel) [reply README.md]: fixed in [8fae996](https://github.com/jpalardy/vim-slime-ext-plugins/pull/6/commits/8fae996d1da07191d1e2453119863ef25d452e92)

Comment (jpalardy) [reply README.md]: it's fine ... I didn't realize :+1:

**Pull request merged**